### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TomintheToon**
Expanded on overview + removed program address table (redundant / misplaced?)

### CODE REVIEW

File: dual-tokenomics/dual-dao/README.md [hunk 1]
```diff
@@ -4,15 +4,12 @@ description: Governance of Dual Finance
 
 # Dual DAO
 
-Deposit DUAL to vote on Dual DAO proposals on [Realms](https://app.realms.today/dao/dual%20dao)\
+Dual DAO is hosted on [Realms](https://app.realms.today/dao/dual%20dao).\
 \
-Treasury earns fees on DIPs and exercised SOs.\
+The DAO enables participants to deposit DUAL tokens in return for elibility to vote on governace proposals. There is no minimum deposit size and there is no lock-up period for the deposit; users can withdraw their tokens at any time.\
```
Comment (jlkramer16) [inline dual-tokenomics/dual-dao/README.md]: Add "In order to vote on proposals, DUAL must be deposited onto realms until proposals are finalized. Voting is currently set to 3 days. Quorum to pass any governance proposal is set to 1% of total token supply & 100,000 tokens are required to create a proposal."

File: dual-tokenomics/dual-dao/README.md [hunk 2]
```diff
@@ -4,15 +4,12 @@ description: Governance of Dual Finance
...
 \
-Participants in governance may be eligible to earn further DUAL Staking Options via Proof-of-Governance. Keep a close eye on Realms and/or the discord [Governance channel](https://discord.com/channels/937797334048325673/1071855278808637560) for proposals
+The Dual DAO has a treasury that includes all of the un-vested DUAL tokens. In addition, the DUAL DAO earns fees on DIPs and exercised SOs.\
+\
+Participants in governance may be eligible to earn further DUAL Staking Options ("Proof-of-Governance Airdrops"). Keep a close eye on Realms and/or the discord [Governance channel](https://discord.com/channels/937797334048325673/1071855278808637560) for upcoming proposals.
 
 <figure><img src="../../.gitbook/assets/image (13).png" alt=""><figcaption></figcaption></figure>
-
-| DiPbvUUJkDhV9jFtQsDFnMEMRJyjW5iS6NMwoySiW8ki - DIP program       |
```
Comment (jlkramer16) [inline dual-tokenomics/dual-dao/README.md:15]: Let's make a separate page to house all program addresses for easy look up

File: dual-tokenomics/dual-dao/README.md [hunk 3]
```diff
@@ -4,15 +4,12 @@ description: Governance of Dual Finance
...
 # Dual DAO
 
-Deposit DUAL to vote on Dual DAO proposals on [Realms](https://app.realms.today/dao/dual%20dao)\
+Dual DAO is hosted on [Realms](https://app.realms.today/dao/dual%20dao).\
 \
-Treasury earns fees on DIPs and exercised SOs.\
+The DAO enables participants to deposit DUAL tokens in return for elibility to vote on governace proposals. There is no minimum deposit size and there is no lock-up period for the deposit; users can withdraw their tokens at any time.\
 \
-Participants in governance may be eligible to earn further DUAL Staking Options via Proof-of-Governance. Keep a close eye on Realms and/or the discord [Governance channel](https://discord.com/channels/937797334048325673/1071855278808637560) for proposals
+The Dual DAO has a treasury that includes all of the un-vested DUAL tokens. In addition, the DUAL DAO earns fees on DIPs and exercised SOs.\
```
Comment (jlkramer16) [inline dual-tokenomics/dual-dao/README.md]: Add screenshot of treasury

File: dual-tokenomics/dual-dao/README.md [hunk 4]
```diff
@@ -4,15 +4,12 @@ description: Governance of Dual Finance
 
 # Dual DAO
 
-Deposit DUAL to vote on Dual DAO proposals on [Realms](https://app.realms.today/dao/dual%20dao)\
+Dual DAO is hosted on [Realms](https://app.realms.today/dao/dual%20dao).\
 \
-Treasury earns fees on DIPs and exercised SOs.\
+The DAO enables participants to deposit DUAL tokens in return for elibility to vote on governace proposals. There is no minimum deposit size and there is no lock-up period for the deposit; users can withdraw their tokens at any time.\
 \
```
Comment (jlkramer16) [inline dual-tokenomics/dual-dao/README.md:10]: Add screenshot of governance parameters: 
image

Comment (TomintheToon) [reply dual-tokenomics/dual-dao/README.md:10]: Added.

File: dual-tokenomics/dual-dao/README.md (hunk 1 above)
Comment (TomintheToon) [reply dual-tokenomics/dual-dao/README.md]: Re-worded with additions as suggested.

File: dual-tokenomics/dual-dao/README.md (hunk 3 above)
Comment (TomintheToon) [reply dual-tokenomics/dual-dao/README.md]: Done.

File: dual-tokenomics/dual-dao/README.md (hunk 2 above)
Comment (TomintheToon) [reply dual-tokenomics/dual-dao/README.md:15]: OK, will give that a go. Suggest it should be added to the Extra section?

Comment (jlkramer16) [reply dual-tokenomics/dual-dao/README.md:15]: Extra works. Can be a seperate Pull request (PR)

 There more now to add:

DIP Smart Contract |  DiPbvUUJkDhV9jFtQsDFnMEMRJyjW5iS6NMwoySiW8ki
SO Smart Contract |  4yx1NJ4Vqf2zT1oVLk4SySBhhDJXmXFt88ncm4gPxtL7
DIP MM Smart Contract |  esdUULhoeFHro46mvkTizMzg38hKBunvErmPuUw6n9p
GSO | DuALd6fooWzVDkaTsQzDAxPGYCnLrnWamdNNTNxicdX8
AIrdrop Program | 2fJcpdR6qzqDP7fBqvoJQ5PGYdaRFBNyUKZkZf5t12mr
Basic Verifier Program | FEdxZUg4BtWvMy7gy7pXEoj1isqBRYmbYdpyZfq5QZYr
Merkle Verifier Program |  8tBcmZAMNm11DuGAS2r6PqSA3CKt72amoz8bVj14xRiT
Password Verifier Program |  EmsREpwoUtHnmg8aSCqmTFyfp71vnnFCdZozohcrZPeL
Governance Verifier Program | ATCsJvzSbHaJj3a9uKTRHSoD8ZmWPfeC3sYxzcJJHTM5

Comment (TomintheToon) [reply dual-tokenomics/dual-dao/README.md:15]: Created PR for new page

Comment (jlkramer16) [reply dual-tokenomics/dual-dao/README.md:15]: Where? link it here

Comment (TomintheToon) [reply dual-tokenomics/dual-dao/README.md:15]: Sorry, I thought it would get flagged to you somehow.

https://github.com/TomintheToon/docs/pull/1

**Pull request merged**